crates/runtime/src/context.rs

```diff
@@ -23,8 +22,9 @@ pub struct Context {
     app_addr: AppAddr,
 }
 
-// SVM is single-threaded.
-// `Send`, `Sync` and `Clone` are required by `wasmer::WasmerEnv`
+// On SAFETY:
```

Comment:
can we use `#SAFETY` instead?

Reply:
Isn't `# Safety` for public APIs that expose `unsafe`? This is an internal comment. Just trying to understand what to do in the future.

Reply:
Actually I don't know 🤷

Reply:
Let's use the same everywhere and not complicate our lives unnecessarily

Reply:
so `#SAFETY` everywhere?

Reply:
Yes imho. Fixing and resolving.

Reply:
Any reason why you don't want the space? Grepping maybe?

Reply:
or `#Safety` everywhere?

Reply:
Let's just do `# Safety` (with a space) which is the one most commonly used rn in the codebase and also what https://rust-lang.github.io/api-guidelines/documentation.html suggests.

Reply:
> Any reason why you don't want the space? Grepping maybe?

I've never seen this convention. I've only encountered `#SAFETY` or `#Safety`.
So I'm not sure what's the right choice. I'll let you decide.

Reply:
Settled on `# Safety`